sdks/java/extensions/protobuf/src/test/proto/payload_messages.proto

```diff
@@ -42,3 +42,18 @@ message SimpleMessage {
   int32 id = 1;
   string name = 2;
 }
+
+message NameMessage {
```

Comment:
possible nit: I'd use less-simple messages, at least one of which has a submessage or repeated field? TestMessage with ItMessage above would work well.